src/app/GitUI/Interops/RECT.cs

```diff
@@ -4,6 +4,7 @@ namespace System
 {
     internal static partial class NativeMethods
     {
+        [StructLayout(LayoutKind.Sequential)]
         public struct RECT
```

Comment:
still there?

> Thus I've linked `RECT.cs` into `GitExtUtils`.

I see two files in commit 7c602bca59c40d4b152142cc7225d5ef91946303 both in GE and on GH.

Reply:
https://github.com/gitextensions/gitextensions/pull/11864#discussion_r1741204729